### PR TITLE
[Deribit] future_combo and options_combo breaking init

### PR DESCRIPTION
**Pull request by mdvx**
fix in 1 PR

### CODE REVIEW

**Comment by hanc1208** [comment]
I think that the indent level is different with the original one.

**Comment by hanc1208** [comment]
ah, thank you. I hope that this PR is merged asap :)

File: xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitAdapters.java [hunk 1]
```diff
@@ -283,8 +284,14 @@ public static Instrument adaptInstrument(String instrumentName) {
     String[] parts = instrumentName.split("-");
     if (parts.length == 2) {
       DeribitInstrument future = new DeribitInstrument();
-      future.setBaseCurrency(parts[0]);
-      future.setQuoteCurrency(IMPLIED_COUNTER);
+      if(parts[0].contains("_")){
```
Comment (earce) [inline xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitAdapters.java:287]: can we fix spacing?

File: xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitExchange.java [hunk 2]
```diff
@@ -23,85 +24,91 @@
...
+        exchangeSpecification.setExchangeDescription("Deribit is a Bitcoin futures exchange");
+        return exchangeSpecification;
+    }
+
+    public ExchangeSpecification getSandboxExchangeSpecification() {
+
+        ExchangeSpecification exchangeSpecification = new ExchangeSpecification(this.getClass());
+        exchangeSpecification.setSslUri("https://test.deribit.com/");
+        exchangeSpecification.setHost("test.deribit.com");
+        //    exchangeSpecification.setPort(80);
```
Comment (earce) [inline xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitExchange.java:57]: here too

**Review by earce — APPROVED** [review]

**Pull request merged**